hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointMetrics.java

```diff
@@ -40,8 +40,10 @@ public class DBCheckpointMetrics {
   // Metrics to track checkpoint statistics from last run.
   private @Metric MutableGaugeLong lastCheckpointCreationTimeTaken;
   private @Metric MutableGaugeLong lastCheckpointStreamingTimeTaken;
+  private @Metric MutableGaugeLong lastCheckpointStreamingNumSSTExcluded;
   private @Metric MutableCounterLong numCheckpoints;
   private @Metric MutableCounterLong numCheckpointFails;
+  private @Metric MutableCounterLong numIncrementalCheckpoints;
```

Comment:
Please add one comment that IC are included in the numCheckpoints too.

Reply:
Good suggestion.